src/components/Article.jsx

```diff
@@ -1 +1,33 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US',
+    {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+    });
+
+  return (
+    <article className="box">
```

Comment:
we have a simplified syntax: 
<>
some text
</>

Reply:
@DenysDubas But here we use semantic? What would be better?